packages/code-morph/src/morphers/tests/definition-schema-wrapper/simple/output.ts

```
@@ -0,0 +1,26 @@
+// @ts-nocheck
```

ESLint

[error] Do not use "@ts-nocheck" because it alters compilation errors.

(@typescript-eslint/ban-ts-comment)